README.md

```diff
@@ -1,36 +1,520 @@
-# Terraform Cloud Operator for Kubernetes (terraform-k8s)
+# Terraform Cloud Operator for Kubernetes (TFCOK)
```

Comment:
I didn't change this since the naming aligned with the repo, and with consul-k8s, vault-k8s

Reply:
Also for an abbreviation, what if we follow the TFC4B branding and make it TFC4K?